cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/vis_builder/vis_types/table.spec.js

```diff
@@ -88,7 +88,7 @@ export const removeBucket = (bucket) => {
 export const testSplitTables = (num) => {
   cy.getElementByTestId('visTable')
     .should('have.class', 'visTable')
-    .find('[class="visTable__group"]')
+    .find('[class*="visTable__group"]')
```

Comment:
Nit: can it simply be `find('.visTable__group')`?

Reply:
Yes, I tested it now and it works
<img width="1512" alt="image" src="https://github.com/opensearch-project/opensearch-dashboards-functional-test/assets/67782303/b96a80c3-7921-411f-83e4-9347ac3b9ce3">
I have updated PR to simpler version you suggested